Vizzuality/gfw
# Feat/recent imagery

## Overview

Integrate the recent imagery feature with the new map. This includes some improvements.
- Dragging batches fetches better
- Tooltip on hover with natuve map events
- GeoJSON paints on the map correctly when passed
- Improved styles for datepickers
- Loader for recent button to show when fetching new tiles

## Demo

![2018-09-06 14 36 52](https://user-images.githubusercontent.com/20288774/45157773-57e2b280-b1e2-11e8-9b47-c3d2d588020c.gif)

## Notes

So extra improvements have been made to the menu and layer management.

## Testing

Go to anywhere on map:
- enable recent imagery
- drag outside box to see tile change
- click on box to see settings menu
- change settings
- deactivate recent imagery
